### PR TITLE
Feature/data fetching

### PR DESCRIPTION
**Pull request by gmbrandt**
Implements remote fetching of the scanning law for Gaia.

### CODE REVIEW

File: htof/parse.py [hunk 1]
```diff
@@ -178,6 +181,108 @@ def __init__(self, scan_angle=None, epoch=None, residuals=None, inverse_covarian
...
+            parallaxFactorAc = details.find('parallaxFactorAc').text
+            observationTimeAtBarycentre = event.find('eventTcbBarycentricJulianDateAtBarycentre').text
+            rows.append([name, raR, decR, raH, decH, observationTimeAtGaia, ccdRow,
+                         zetaFieldAngle, scanAngle, fov, parallaxFactorAl, parallaxFactorAc, observationTimeAtBarycentre])
+        data = pd.DataFrame(rows, columns=columns)
+        data = data.astype({"Target": str,"ra[rad]": float, "dec[rad]": float,"ra[h:m:s]": str,"dec[d:m:s]": str,"ObservationTimeAtGaia[UTC]": str,"CcdRow[1-7]": int,"zetaFieldAngle[rad]": float,"scanAngle[rad]": float,"Fov[FovP=preceding/FovF=following]": str,"parallaxFactorAlongScan": float,"parallaxFactorAcrossScan": float,"ObservationTimeAtBarycentre[BarycentricJulianDateInTCB]": float })
+        return data
+    
+    def keep_field_hits(self, data):
+        format = "%Y-%m-%dT%H:%M:%S.%f"
```
Comment (gmbrandt) [inline htof/parse.py:245]: This function would benefit from a doc string. Something like 
"""
Gost files downloaded from the web contain sequences of ten observations, for every observation of the star in the scanning law. The first entry is the skymapper CCD hit, and the extra entries are redundant (the hits for astrometric field CCD's 1 through 9). Only the second observation of each sequence should be saved. This function saves the second observation (this is the hit on the first astrometric field CCD (AF1)).
"""

**Review by gmbrandt — COMMENTED** [review]
PR looks good, just needs one additional docstring to the keep_field_hits method.

File: htof/parse.py (hunk 1 above)
Comment (gavinkhung) [reply htof/parse.py:249]: I agree, I have added the doc string.

**Review by gavinkhung — COMMENTED** [review]
Pull Request looks good to me.

**Review by gavinkhung — APPROVED** [review]

**Pull request merged**